TheDescent/Esd109

Ethan-Dav109 opened the pull request:
Intro to Github Assignment

bjones1 commented on TheDescent/esd109/The DescentESD109_updated.py:
```diff
@@ -0,0 +1,94 @@
+# # Ethan Davis ESD109 The Descent CodeChat Editor
```
What is this? I want just one code file with the solution, not two.

bjones1 replied:
This looks like you resolved it -- click the "Resolve conversation" below so you can keep track of what's done and what still needs doing.

Ethan-Dav109 commented:
Updated it, I do not know what in the world those random numbers and symbols are, they look like they are being added in while it transfers. I removed the second file and updated the first one.
________________________________
From: Bryan A. Jones ***@***.***>
Sent: Tuesday, October 15, 2024 4:24 PM
To: bjones1/literate-programming-github-fall-2024 ***@***.***>
Cc: Davis, Ethan ***@***.***>; Author ***@***.***>
Subject: Re: [bjones1/literate-programming-github-fall-2024] TheDescent/Esd109 (PR #42)


@bjones1 requested changes on this pull request.

________________________________

In TheDescent/esd109/The DescentESD109_updated.py<https://github.com/bjones1/literate-programming-github-fall-2024/pull/42#discussion_r1802036959>:

> @@ -0,0 +1,94 @@
+# # Ethan Davis ESD109 The Descent CodeChat Editor


What is this? I want just one code file with the solution, not two.

—
Reply to this email directly, view it on GitHub<https://github.com/bjones1/literate-programming-github-fall-2024/pull/42#pullrequestreview-2370730232>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BK2QZ65IBS4EUDDS5NWS6M3Z3WB2TAVCNFSM6AAAAABP7H7QRCVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDGNZQG4ZTAMRTGI>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

bjones1 commented on TheDescent/esd109/The DescentESD109.py:
```diff
@@ -1,34 +1,44 @@
+# when I first began to solve this problem.\
+# \
+#
+# <span style="background-color: #2dc26b;">esd109:</span> Thank you for this
```
I still see these in the file...
